python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -57,6 +57,7 @@
 
 
 class RasterizeAlgorithm(QgisAlgorithm):
+
```

Comment:
Looking at all the PEP8 sources, I can't find any reference to this spacing. Was that created by a script?